[ERROR] Execution count overflow detected.

I'm getting `[ERROR] Execution count overflow detected.` when running with this setup:

```
rustup component add llvm-tools-preview
RUSTFLAGS="-Zinstrument-coverage"
QUICKCHECK_TESTS=1
cargo +nightly build --all-features --workspace
LLVM_PROFILE_FILE="test-%p-%m.profraw" cargo +nightly test --all-features --workspace
grcov --binary-path ./target/debug/ -s . -t html --branch --ignore-not-existing --ignore "/*" -o ./coverage/ .
```

Also sometimes, trying different ways of making `grcov` work, I'm getting `[ERROR] FN 'ink_env::backend::ReturnFlags::set_reverted' duplicated for 'crates/env/src/backend.rs' in a lcov file`

Facing the same issue:

```
// Darwin Kernel Version 19.6.0

$ rustc 1.50.0 (cb75ad5db 2021-02-10)
$ grcov 0.7.1
$ 21:54:06 [ERROR] Execution count overflow detected
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -56,7 +56,7 @@ pub mod html;
 mod file_filter;
 pub use crate::file_filter::*;
 
-use log::error;
+use log::{error, warn};
 use std::collections::{btree_map, hash_map};
 use std::fs;
 use std::io::{BufReader, Cursor};
@@ -142,7 +142,7 @@ fn add_results(
     }
 
     if warn_overflow {
-        error!("Execution count overflow detected.");
+        warn!("Execution count overflow detected.");
     }
 }
 

